scripts/ci/_utils.sh

```diff
@@ -227,6 +227,9 @@ function initialize_common_environment {
 
     # Artifact name suffix for SVN packaging
     export VERSION_SUFFIX_FOR_SVN=""
+
+    # Whether base python images should be pulled from cache
+    export PULL_PYTHON_BASE_IMAGES_FROM_CACHE="true"
```

Comment:
Won't this overwrite the value set by the ci.yml file?

Reply:
```suggestion
    export PULL_PYTHON_BASE_IMAGES_FROM_CACHE="${PULL_PYTHON_BASE_IMAGES_FROM_CACHE:="true"}"
```

no?

Reply:
Indeed good catch. It was wrong place as well I moved it higher next to push images where it belongs.